scripts/format-makefile-am.pl

```diff
@@ -19,7 +19,9 @@
     }
     # accumulate files and prep for sorting
     my %files = ();
-    if (/^(\S+_SOURCES)\s*(\+?=)\s*(.*[^\\])$/ ) {
+    # TODO: Handle or rename /\S+SOURCE=/ and /\S*[^_]SOURCES=/
```

Comment:
This TODO deserves a dedicated PR. It is not directly related to the problem solved in this PR, it requires some non-trivial work to identify all applicable group names, and it will trigger a lot of Makefile.am changes.

Reply:
FTR; The *_SOURCES tag is a reserved keyword by automake. It has logic side effects during build.